Dockerfile

```diff
@@ -35,7 +35,14 @@ RUN pip install -r requirements-full.txt && \
 # https://github.com/joblib/joblib/blob/0.11/joblib/parallel.py#L328L342
 ENV JOBLIB_TEMP_FOLDER=/tmp
 
-ENV VERSION=7.0.0 \
+ENV VERSION=7.1.0 \
   VERSION_MAJOR=7 \
-  VERSION_MINOR=0 \
+  VERSION_MINOR=1 \
   VERSION_MICRO=0
+
+FROM production AS test
+COPY .circleci/test_image.py .
+COPY CHANGELOG.md .
```

Comment:
why is the changelog.md file needed?

Reply:
`CHANGELOG.md` is for the test that verifies if the latest version number in the changelog matches the environment variable `VERSION` in the image. It's too easy for us to forget updating one of these things.

Reply:
nice!! i like this. im gunna have to steal this for some of our repos